From: KevinDuan <duankaiwen@lingmoos.cn>
Date: Tue, 23 Jul 2024 09:28:47 +0000
Subject: Translated using Weblate (Uyghur)

Currently translated at 100.0% (17 of 17 strings)

Translation: lingmo-nile-new/lingmo-search-appwidget
Translate-URL: http://weblate.lingmo.top/projects/lingmo-nile-new/lingmo-search-appwidget/ug/
---
 translations/lingmo-search/appwidget/search_kk_KZ.ts | 34 +++++++++++-----------
 translations/lingmo-search/appwidget/search_ug.ts    | 34 +++++++++++-----------
 2 files changed, 34 insertions(+), 34 deletions(-)

diff --git a/translations/lingmo-search/appwidget/search_kk_KZ.ts b/translations/lingmo-search/appwidget/search_kk_KZ.ts
index 5ad317b..fc10af6 100644
--- a/translations/lingmo-search/appwidget/search_kk_KZ.ts
+++ b/translations/lingmo-search/appwidget/search_kk_KZ.ts
@@ -5,86 +5,86 @@
     <name>LingmoSearch::BestListWidget</name>
     <message>
         <source>Best Matches</source>
-        <translation type="unfinished"></translation>
+        <translation>ەڭ جاقسى سايكەستىرۋ</translation>
     </message>
 </context>
 <context>
     <name>LingmoSearch::CreateIndexAskDialog</name>
     <message>
         <source>lingmo-search</source>
-        <translation type="unfinished"></translation>
+        <translation>ٸزدەۋ lingmo</translation>
     </message>
     <message>
         <source>Search</source>
-        <translation type="unfinished"></translation>
+        <translation>ٸزدەۋ</translation>
     </message>
     <message>
         <source>close</source>
-        <translation type="unfinished"></translation>
+        <translation>جابۋ</translation>
     </message>
     <message>
         <source>Creating index can help you get results more quickly. Would you like to create one?</source>
-        <translation type="unfinished"></translation>
+        <translation>كورسەتكىش تاقپٸرلاۋ ەتۋ الٸدە تەز ناتيجەگە ئېرىشىشىڭىزگە جاردەم بەرەدٸ. بٸرٸن تاقپٸرلاۋ ەتۋ ورىندايسىزبا؟</translation>
     </message>
     <message>
         <source>Don&apos;t remind me again</source>
-        <translation type="unfinished"></translation>
+        <translation>جۇرەيىك قاتە ەسكەرتپە</translation>
     </message>
     <message>
         <source>No</source>
-        <translation type="unfinished"></translation>
+        <translation>كۇشىنەن قالتۇرۇش</translation>
     </message>
     <message>
         <source>Yes</source>
-        <translation type="unfinished"></translation>
+        <translation>تۇراقتاندىرۋ</translation>
     </message>
 </context>
 <context>
     <name>LingmoSearch::MainWindow</name>
     <message>
         <source>lingmo-search</source>
-        <translation type="unfinished"></translation>
+        <translation>ٸزدەۋ lingmo</translation>
     </message>
 </context>
 <context>
     <name>LingmoSearch::SearchLineEdit</name>
     <message>
         <source>Search</source>
-        <translation type="unfinished"></translation>
+        <translation>ٸزدەۋ</translation>
     </message>
 </context>
 <context>
     <name>LingmoSearch::LingmoSearchGui</name>
     <message>
         <source>Quit lingmo-search application</source>
-        <translation type="unfinished"></translation>
+        <translation>ٸزدەۋ ئەپتىن شەگىنۋ lingmo</translation>
     </message>
     <message>
         <source>Show main window</source>
-        <translation type="unfinished"></translation>
+        <translation>نەگٸزگٸ تەرەزەسى كورسەتۋ</translation>
     </message>
     <message>
         <source>unregister a plugin with &lt;pluginName&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>نى ٸستەتۋ &lt;pluginName&gt; تٸزٸمدەن ٴوشىرۋ قىستىرما دەتالى</translation>
     </message>
     <message>
         <source>register a plugin with &lt;pluginName&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>نى ٸستەتۋ &lt;pluginName&gt; تٸزٸمدەۇ قىستىرما دەتالى</translation>
     </message>
     <message>
         <source>move &lt;pluginName&gt; to the target pos</source>
-        <translation type="unfinished"></translation>
+        <translation>نى نىسانالىق ورٸنعا جوتكەۋ &lt;pluginName&gt;</translation>
     </message>
     <message>
         <source>move plugin to &lt;index&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>عا جوتكەۋ &lt;index&gt; قىستىرما دەتالدى</translation>
     </message>
 </context>
 <context>
     <name>search</name>
     <message>
         <source>search</source>
-        <translation>іздеу</translation>
+        <translation>ٸزدەۋ</translation>
     </message>
 </context>
 </TS>
diff --git a/translations/lingmo-search/appwidget/search_ug.ts b/translations/lingmo-search/appwidget/search_ug.ts
index a283313..61f3a84 100644
--- a/translations/lingmo-search/appwidget/search_ug.ts
+++ b/translations/lingmo-search/appwidget/search_ug.ts
@@ -5,86 +5,86 @@
     <name>LingmoSearch::BestListWidget</name>
     <message>
         <source>Best Matches</source>
-        <translation type="unfinished"></translation>
+        <translation>ئەڭ ياخشى ماسلاشتۇرۇش</translation>
     </message>
 </context>
 <context>
     <name>LingmoSearch::CreateIndexAskDialog</name>
     <message>
         <source>lingmo-search</source>
-        <translation type="unfinished"></translation>
+        <translation>ئىزدەش lingmo</translation>
     </message>
     <message>
         <source>Search</source>
-        <translation type="unfinished"></translation>
+        <translation>ئىزدەش</translation>
     </message>
     <message>
         <source>No</source>
-        <translation type="unfinished"></translation>
+        <translation>ئەمەلدىن قالتۇرۇش</translation>
     </message>
     <message>
         <source>Yes</source>
-        <translation type="unfinished"></translation>
+        <translation>جەزملەشتۈرۈش</translation>
     </message>
     <message>
         <source>close</source>
-        <translation type="unfinished"></translation>
+        <translation>يېپىش</translation>
     </message>
     <message>
         <source>Creating index can help you get results more quickly. Would you like to create one?</source>
-        <translation type="unfinished"></translation>
+        <translation>ئىندېكس بەرپا قىلىش تېخىمۇ تېز نەتىجىگە ئېرىشىشىڭىزگە ياردەم بېرىدۇ. بىرنى بەرپا قىلىش قىلامسىز؟</translation>
     </message>
     <message>
         <source>Don&apos;t remind me again</source>
-        <translation type="unfinished"></translation>
+        <translation>ماڭا قايتا ئەسكەرتمەڭ</translation>
     </message>
 </context>
 <context>
     <name>LingmoSearch::MainWindow</name>
     <message>
         <source>lingmo-search</source>
-        <translation type="unfinished"></translation>
+        <translation>ئىزدەش lingmo</translation>
     </message>
 </context>
 <context>
     <name>LingmoSearch::SearchLineEdit</name>
     <message>
         <source>Search</source>
-        <translation type="unfinished"></translation>
+        <translation>ئىزدەش</translation>
     </message>
 </context>
 <context>
     <name>LingmoSearch::LingmoSearchGui</name>
     <message>
         <source>Quit lingmo-search application</source>
-        <translation type="unfinished"></translation>
+        <translation>ئىزدەش ئەپتىن چېكىنىش lingmo</translation>
     </message>
     <message>
         <source>Show main window</source>
-        <translation type="unfinished"></translation>
+        <translation>ئاساسىي كۆزنەكنى كۆرسىتىش</translation>
     </message>
     <message>
         <source>unregister a plugin with &lt;pluginName&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>نى ئىشلىتىش &lt;pluginName&gt; تىزىمدىن ئۆچۈرۈش قىستۇرما دېتالى</translation>
     </message>
     <message>
         <source>register a plugin with &lt;pluginName&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>نى ئىشلىتىش &lt;pluginName&gt; تىزىملىتىش قىستۇرما دېتالى</translation>
     </message>
     <message>
         <source>move &lt;pluginName&gt; to the target pos</source>
-        <translation type="unfinished"></translation>
+        <translation>نى نىشانلىق ئورۇنغا يۆتكەش &lt;pluginName&gt;</translation>
     </message>
     <message>
         <source>move plugin to &lt;index&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>غا يۆتكەش &lt;index&gt; قىستۇرما دېتالنى</translation>
     </message>
 </context>
 <context>
     <name>search</name>
     <message>
         <source>search</source>
-        <translation type="unfinished"></translation>
+        <translation>ئىزدەش</translation>
     </message>
 </context>
 </TS>
